SPECS/containerized-data-importer/containerized-data-importer.spec

```diff
@@ -18,13 +18,14 @@
 Summary:        Container native virtualization
 Name:           containerized-data-importer
 Version:        1.55.0
-Release:        10%{?dist}
+Release:        11%{?dist}
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System/Packages
 URL:            https://github.com/kubevirt/containerized-data-importer
 Source0:        https://github.com/kubevirt/containerized-data-importer/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Patch0:         allow-dynamic-build.patch
```

Comment:
In order to apply the patch, you'll also need `%setup` to take it into account. For this spec the easiest way would probably be to add a `%patch0 -p1` line somewhere under `%setup`, but I also encourage you to try switching to `%autosetup -p1`